### ISSUE TITLE
Windows: selected tree item not accessible

### ISSUE THREAD

**Issue by chrmarti** [Microsoft/vscode#13104]
- VSCode Version: Code - Insiders 1.6.0-insider (c96822698cdbd47b8245fb5f9f45a0168bc9da33, 2016-09-30T17:37:14.267Z)
- OS Version: Windows_NT ia32 10.0.14393
- Extensions: `dbaeumer.vscode-eslint-1.0.7`, `eg2.tslint-0.5.39`

Steps to Reproduce:
1. Use nvda for accessibility testing
2. Open Command Palette with keyboard shortcut
3. Type something
4. Use arrow keys to navigate list
5. Observe: nvda keeps re-reading the entered text, not the selected item which makes it inaccessible for nvda users

**Comment by bpasero** [comment]
Yup, electron update.

**Comment by bpasero** [comment]
I verified that this will be fixed in Chrome 53+.
